mp/src/materialsystem/stdshaders/pbr_dx9.cpp

```diff
@@ -68,6 +79,10 @@ BEGIN_VS_SHADER(PBR, "PBR shader")
 		SHADER_PARAM(BUMPMAP, SHADER_PARAM_TYPE_TEXTURE, "", "Normal texture");
         SHADER_PARAM(USEENVAMBIENT, SHADER_PARAM_TYPE_BOOL, "0", "Use the cubemaps to compute ambient light.");
 		SHADER_PARAM(SPECULARTEXTURE, SHADER_PARAM_TYPE_TEXTURE, "", "Specular F0 RGB map");
+#ifdef PBRPARALLAX
```

Comment:
Why does the parallax need to be IfDeffed?

Reply:
Habit... ill remove it if you desire not to have them